bin/webpack-dev-server.js

```diff
@@ -390,8 +390,9 @@ function startDevServer(webpackOptions, options) {
 
   ['SIGINT', 'SIGTERM'].forEach((sig) => {
     process.on(sig, () => {
-      server.close();
-      process.exit(); // eslint-disable-line no-process-exit
+      server.close(() => {
```

Comment:
`process.nextTick()` ? But not 💯

Reply:
@michael-ciniawsky hm? please elaborate :)

Reply:
Hmm I think it is bananas from my side tbh :), the intend was 

```js
server.close()

process.nextTick(() => process.exit())
```

to ensure `server.close()` finishes before `process.exit()` is called by pushing it to the next EventLoop 'tick'.  But a callback to `server.close()` is definitely cleaner and more expressive/logical, so forget what I said 😛

Reply:
haha no worries. something good to keep in mind though!